### ISSUE TITLE
Take Home Assignment Figma File

### ISSUE THREAD

**Issue by sabrinang** [MozillaFoundation/Design#1213]
Create a figma file that we will share with candidates for their take home assignment test.

**Comment by sabrinang** [comment]
Worked on revising file from design crit and jam feedback. 

Take home test figma file here: https://www.figma.com/file/bMn04IOgEp45YBNg8gqf1j/Take-Home-Test-for-UX%2FUI-Designer-(2023)?type=design&node-id=0%3A1&t=8JJ3GDpc6pgNkGEK-1

Currently, figuring out library share settings but leaning towards importing styles and copy/paste components.

**Comment by sabrinang** [comment]
@nancyt1 https://www.figma.com/file/bMn04IOgEp45YBNg8gqf1j/Take-Home-Test-for-UX%2FUI-Designer-(2023)?type=design&node-id=96%3A3865&mode=design&t=u16HDTOK0Us2pdX1-1

I've continued revising the file and tested out how it works when duplicated/shared with a non team member.

As a result, I imported styles from token studio, copied over grid styles and created a "Components" page with copied over components locally. This is because since the interviewees are not in on our Figma team/organization, I discovered they will be able to see our published team libraries as I see the file appears to have ‘missing libraries’ and no components in asset panel nor local styles when viewed from my personal figma account.

Ready for your review and would appreciate your thoughts on this!
